### DISCUSSION TITLE
App wont load when I preview or view it on any device.

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#2600] [Help]
I went to look at the preview of my app to test something and I was surprised with a white screen with a blue loading bar at the top. I've waited for it to load for a while and nothing happened. Not sure what the issue is.

**Comment by mjashanks** [comment]
Hi @PixelParallax  - could you 
1. open up your dev tools (`F12` on Chrome)
2. click on the console tab
3. Refresh the page

Can you let me know if you see any errors in there - and paste them in here.

Thanks.

**Reply by adamsanclemente** [reply]
Yea theres a few: Heres the errors in the preivew window:
```
GET http://209.97.155.21:10000/prod-budi-app-assets/app_e48e0ff7778f42c391e8deef12f0406a/budibase-client.js net::ERR_ABORTED 404 (Not Found)
app_dev_e48e0ff7778f42c391e8deef12f0406a:27 Uncaught ReferenceError: loadBudibase is not defined
    at app_dev_e48e0ff7778f42c391e8deef12f0406a:27
(anonymous) @ app_dev_e48e0ff7778f42c391e8deef12f0406a:27
favicon.ico:1 GET http://209.97.155.21:10000/favicon.ico 403 (Forbidden)
```

Then now the design window wont load heres the errors for that:
```
GET http://209.97.155.21:10000/prod-budi-app-assets/app_e48e0ff7778f42c391e8deef12f0406a/budibase-client.js net::ERR_ABORTED 404 (Not Found)
app_dev_e48e0ff7778f42c391e8deef12f0406a:27 Uncaught ReferenceError: loadBudibase is not defined
    at app_dev_e48e0ff7778f42c391e8deef12f0406a:27
(anonymous) @ app_dev_e48e0ff7778f42c391e8deef12f0406a:27
favicon.ico:1 GET http://209.97.155.21:10000/favicon.ico 403 (Forbidden)
```

**Reply by mjashanks** [reply]
Thanks for that. It looks like you app is missing of the built-in libraries. Could you try export our app then importing. This should create a brand new app, and hopefully pull the library in with it
